PeerioTechnologies/peerio-desktop
# fix: set default peerioContentEnabled to false

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/2721/desktop-unable-to-disable-image-previews
 
#### Testing instructions  
Images should not load when "Not now" or "Disabled" selected from Peerio content consent, should load when "Enabled" selected.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
